src/website/src/app/documentation/demos/datagrid/filtering/examples.ts

```diff
@@ -118,16 +118,23 @@ class ColorFilter implements ClrDatagridFilterInterface<User> {
 </clr-dg-column>
 `,
   filterSearchResults: `
-<-- 
-  Will search into user.creation that in this case is 'Date' object
-  and not the result of the pipe 'Jan 6, 2018' 
--->
-<clr-dg-cell [clrDgField]="'creation'">{{ user.creation | date }}</clr-dg-cell>
+<clr-datagrid>
+    <-- 
+      This will search into user.creation (a Date object) and not the result
```

Comment:
The comment here refers to "the result of the pipe", which you already removed.
I suggest adding back the 
"<clr-dg-cell ~[clrDgField]="'creation'"~>{{ user.creation | date }}</clr-dg-cell>"
after the `clr-dg-column` definition. It's okay to have both lines, as long as they are correct.

Reply:
I made this code snippet a bit longer so that it includes two column definitions (with the comments) as well as the row definitions below (which includes the date pipe).  Does that make sense or is the row part too far away?

Reply:
@Jinnie Just following up -- does that make sense?  I could also move the comments above the datagrid definition if you think that would make it easier to read.